mw_jwt.go

```diff
@@ -456,9 +495,8 @@ func (k *JWTMiddleware) ProcessRequest(w http.ResponseWriter, r *http.Request, _
 	if err != nil {
 		logEntry.Error("JWT validation error: ", err)
 		return errors.New("Key not authorized:" + err.Error()), http.StatusForbidden
-	} else {
-		return errors.New("Key not authorized"), http.StatusForbidden
 	}
+	return errors.New("Key not authorized"), 403
```

Comment:
I think it is better to use constants like `http.StatusForbidden` for magic http-codes, easier to read and reduces number of bugs caused by typos